tests/NHapi.NUnit.SourceGeneration/NHapi.NUnit.SourceGeneration.csproj

```diff
@@ -26,8 +26,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
-    <PackageReference Include="Verify.NUnit" Version="19.5.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
+    <PackageReference Include="Verify.NUnit" Version="19.3.0" />
```

Comment:
Was this version intentionally lowered (19.5.0 -> 19.3.0)?

Reply:
nope this was an accident, should now be fixed.